.github/workflows/release.yml

```
@@ -30,10 +30,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - env:
+        HEAD_REF: ${{ github.head_ref }}
       - run: >
           gh release create --target ${{
           github.event.pull_request.merge_commit_sha }} {--title=CML\
-          ,}$(basename ${{ github.head_ref }}) --generate-notes --draft
+          ,}$(basename $HEAD_REF) --generate-notes --draft
```

```suggestion
          ,}"$(basename "$HEAD_REF")" --generate-notes --draft
```

Without proper quoting, the suggested changes would be ineffective as well.

---

https://github.com/iterative/cml/pull/1448/commits/35cfca563ee885ff520226cf50f69891a41fc255